apps/design-system/registry/default/example/admonition-demo.tsx

```diff
@@ -1,11 +1,26 @@
+import Link from 'next/link'
+import { Button } from 'ui'
 import { Admonition } from 'ui-patterns/admonition'
 
 export default function AdmonitionDemo() {
   return (
     <Admonition
       type="default"
-      title="Is it accessible?"
-      description="Yes. It adheres to the WAI-ARIA design pattern."
+      layout="horizontal"
+      className="mb-12 [&>div]:!translate-y-0"
```

Comment:
This `[&>div]:!translate-y-0` has been marked for holistic cleanup in future PR. Right now it’s a common pattern to optically center the contents.